lib/meilisearch/client.rb

```diff
@@ -31,17 +31,14 @@ def indexes(options = {})
 
     # Usage:
     # client.create_index('indexUID')
-    # client.create_index('indexUID', primaryKey: 'id')
-    def create_index(index_uid, options = {})
+    # client.create_index('indexUID', { primaryKey: 'id' })
+    # client.create_index('indexUID', { primaryKey: 'id' }, wait: true)
+    def create_index(index_uid, options = {}, wait: false)
```

Comment:
This change has breaking change about passing arguments.
I think that passing keyword arguments to argument of hash is possible but deprecated ruby3 and above.
https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

Reply:
Changed to keep hash definition for compatibility